src/utils/storage.js

```diff
@@ -0,0 +1,60 @@
+/*
+  Basic file functions scoped under HOMEDIR/.mashery-toolbelt directory
+  Exports: readFileSync, writeFileSync, unlinkSync functions
+
+  Example:
+    writeFileSync('/data/services/1.json', someData)
+    will create file HOMEDIR/.mashery-toolbelt/data/services/1.json
```

Comment:
any ideas where this will be on windows? It would be good to document it in the README..

Reply:
I use native `require('os').homedir()` function. Which by googling should point to `/Users/NAME`